### PR TITLE
Fix error display in parallel mode

### PR DESCRIPTION
**Pull request by fdubost**
In parallel mode, when a step throw an error, we only see `{}` because error instances aren't correctly stringify in JSON message.

With this PR, error instance are formatted like in `event_protocol_formatter.js`.

### CODE REVIEW

File: src/runtime/parallel/slave.js [hunk 1]
```diff
@@ -26,12 +28,26 @@ export default class Slave {
...
     EVENTS.forEach(name => {
-      this.eventBroadcaster.on(name, data =>
-        this.stdout.write(
+      this.eventBroadcaster.on(name, data => {
+        // Format error instance
+        if (data.result && data.result.exception instanceof Error) {
+          data.result.exception = data.result.exception.stack.replace(
+            this.pathRegexp,
+            ''
+          )
```
Comment (charlierudolph) [inline src/runtime/parallel/slave.js]: I don't like replacing the error with just the stack as it loses other fields set on the object.

File: src/runtime/parallel/slave.js [hunk 2]
```diff
@@ -26,12 +28,26 @@ export default class Slave {
     this.cwd = cwd
     this.eventBroadcaster = new EventEmitter()
     this.stackTraceFilter = new StackTraceFilter()
+
+    const pathSepRegexp = new RegExp(escapeStringRegexp(path.sep), 'g')
+    const pathToRemove =
+      cwd.replace(pathSepRegexp, path.posix.sep) + path.posix.sep
+    this.pathRegexp = new RegExp(escapeStringRegexp(pathToRemove), 'g')
```
Comment (charlierudolph) [inline src/runtime/parallel/slave.js]: This path cleanup should go in the formatters and should be its own PR. I'd prefer we replace the cwd with `.` instead of removing it entirely

**Comment by charlierudolph** [comment]
Thanks for your contribution! I ended up going with #1053. I'll make a release today so please let me know if that solves your problem

**Comment by fdubost** [comment]
It is OK, thanks 👍
